NOTICE

```diff
@@ -0,0 +1,19 @@
+The OpenTitan Project
+Copyright 2024 lowRISC contributors.
+
+This product includes hardware and/or software developed as part of the
+OpenTitan(R) project (https://www.opentitan.org,
+https://github.com/lowRISC/opentitan).
+
+-
+
+The Ibex Project
+Copyright 2024 lowRISC contributors.
+
+This product includes hardware and/or software developed as part of the
+Ibex(R) (https://github.com/lowRISC/ibex) and OpenTitan(R) projects.
+
+Ibex was originally developed by the PULP team at ETH Zürich and University of
+Bologna under the name zero-riscy. Ibex verification, performance enhancement
+and security hardening have been supported by the OpenTitan project
```

Comment:
regarding "performance enhancement[s]", I guess we also added some features? (and might add more in the future)

Reply:
Yes but I don't think this necessarily needs to be comprehensive. I think the three categories listed cover most new features.